db/migrate/20160930150810_add_smart_proxy_download_policy.rb

```diff
@@ -0,0 +1,14 @@
+class AddSmartProxyDownloadPolicy < ActiveRecord::Migration
+  def up
+    #set default to on_demand, but update existing proxies to inherit
+    add_column :smart_proxies, :download_policy, :string, :default => ::Runcible::Models::YumImporter::DOWNLOAD_ON_DEMAND
```

Comment:
I think it would be better if the default is what ever `Setting[:default_download_policy]` is?